### ISSUE TITLE
Exception: builtins.KeyError: 'event'

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#169]
```
2017-08-08T11:30:09-0700 [twisted.python.log#info] "127.0.0.1" Po-Boy - [08/Aug/2017:18:30:09 +0000] "GET /ims/api/incident_reports/?event=;incident= HTTP/1.0" 200 494 "https://khaki-shack.prettyok.net/i\
ms/app/events/Test/incidents/new" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13) AppleWebKit/604.1.31 (KHTML, like Gecko) Version/11.0 Safari/604.1.31"
2017-08-08T11:30:12-0700 [ims.application._klein#critical] Request failed
        Traceback (most recent call last):
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 126, in execute_endpoint
            return endpoint_f(self._instance, *args, **kwargs)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 211, in _f
            return _call(instance, f, request, *a, **kw)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 51, in _call
            result = ensureDeferred(result)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 908, in ensureDeferred
            return _inlineCallbacks(None, coro, Deferred())
        --- <exception caught here> ---
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 1386, in _inlineCallbacks
            result = g.send(result)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/application/_api.py", line 260, in newIncidentResource
            incident = modelObjectFromJSONObject(json, Incident)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/model/json/_json.py", line 61, in modelObjectFromJSONObject
            return jsonDeserialize(object, modelClass)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/cattr/converters.py", line 138, in structure
            if not isinstance(cl, _Union)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/cattr/converters.py", line 130, in <lambda>
            self._structure.register(cl, lambda t, o: func(o, t))
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/model/json/_incident.py", line 217, in deserializeIncident
            for key in IncidentJSONKey
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/model/json/_incident.py", line 217, in <genexpr>
            for key in IncidentJSONKey
        builtins.KeyError: 'event'
```

**Comment by wsanchez** [comment]
I think this happens when the web client tries to create a new incident, because the JS doesn't know that the `event` key is now (a required) part of `Incident`.

**Comment by wsanchez** [comment]
Fixed in #175

**Comment by wsanchez** [comment]
Merged
